public/sass/components/_sidemenu.scss

```diff
@@ -84,6 +84,65 @@ $mobile-menu-breakpoint: md;
   top: auto !important;
 }
 
+// nested menus
+.sidemenu-item.dropdown {
+  > ul {
+    li {
+      // the holder
+      position: relative;
+      ul {
+        // the dropdown
+        visibility: hidden;
+        opacity: 0;
+        padding: 0;
+        min-width: 140px;
+        width: auto;
+        position: absolute;
+        top: 0;
+        left: 100%;
+        // the dropdown items style
+        li {
+          margin: 0;
+          display: flex;
+          align-items: center;
+          justify-content: flex-start;
+          height: 30px;
+          a {
+            white-space: nowrap;
+            width: 100%;
+          }
+          // lets put an arrow right
+          // to the inner li\`s with
+          // dropdowns
+          &::before {
+            width: 0;
+            height: 0;
+            border-top: 5px solid transparent;
+            border-bottom: 5px solid transparent;
+            border-left: 5px solid #ffa500;
```

Comment:
Is there any grafana color that can be used here? There are SASS variables in _variables files.

Reply:
Oh, this is garbage code and need to be removed, thanks for pointing!